v2/go.mod

```diff
@@ -3,11 +3,11 @@ module github.com/spiffe/go-spiffe/v2
 go 1.13
```

Comment:
update version to 1.18 here too

Reply:
same for [workflow](https://github.com/spiffe/go-spiffe/blob/main/.github/workflows/pr_build.yaml#L6)